test/tests.py

```diff
@@ -103,54 +103,50 @@ def setup_authentication(self, auth_type, username, password):
             return aws_auth
         return None
 
+    def does_index_match_ignored_index(self, index_name: str):
+        for prefix in self.index_prefix_ignore_list:
+            if index_name.startswith(prefix):
+                return True
+        return False
+
     def set_common_values(self):
-        self.index = f"test_index_{self.unique_id}"
-        self.doc_id = '7'
-        self.ignore_list = []
+        self.index_prefix_ignore_list = ["test_", ".", "searchguard", "sg7", "security-auditlog"]
 
     def setUp(self):
         self.set_common_values()
-        retry_request(delete_index, args=(self.proxy_endpoint, self.index, self.source_auth, self.source_verify_ssl),
-                      expected_status_code=HTTPStatus.NOT_FOUND)
-        retry_request(delete_document, args=(self.proxy_endpoint, self.index, self.doc_id, self.source_auth,
-                                             self.source_verify_ssl),
-                      expected_status_code=HTTPStatus.NOT_FOUND)
-
-    def tearDown(self):
-        delete_index(self.proxy_endpoint, self.index, self.source_auth, self.source_verify_ssl)
-        delete_document(self.proxy_endpoint, self.index, self.doc_id, self.source_auth, self.source_verify_ssl)
 
     def test_0001_index(self):
         # This test will verify that an index will be created (then deleted) on the target cluster when one is created
         # on the source cluster by going through the proxy first. It will verify that the traffic is captured by the
         # proxy and that the traffic reaches the source cluster, replays said traffic to the target cluster by the
         # replayer.
 
+        index_name = f"test_0001_{self.unique_id}"
```

Comment:
I very much appreciate this change so that concurrent build jobs can be running the same test without causing any conflicts.